node_modules.service

```diff
@@ -1,5 +1,15 @@
 <service name="node_modules">
   <summary>download node modules</summary>
   <description>download node modules</description>
+  <parameter name="cpio">
+    <description>cpio file name to store all tarballs in</description>
+  </parameter>
+  <parameter name="output">
+    <description>write rpm source lines to that file</description>
+  </parameter>
+  <parameter name="source-offset">
+    <description>rpm source number to start with</description>
```

Comment:
I think that this can be explained better? Not here but in the doc, just when is used in the first example. I do not have any idea of the implications of the number given in the example.